.redocly.lint-ignore.yaml

```diff
@@ -3,3 +3,5 @@
 openapi.yaml:
   no-empty-servers:
     - '#/servers'
+  spec:
```

Comment:
Note: add lint rule, allowing for nullable field without type specified (metadata can have any type)